packages/block-library/src/comment-template/edit.js

```diff
@@ -55,7 +114,13 @@ export default function CommentTemplateEdit( {
 	);
 
 	const blockContexts = useMemo(
-		() => comments?.map( ( comment ) => ( { commentId: comment.id } ) ),
+		() =>
+			convertToTree(
```

Comment:
It's the only usage of `convertToTree` so maybe we can avoid calling `map` here and handle everything based on the `id` property? It looks like unnecessary step in the process.

Reply:
Right, makes sense 👍

Reply:
Done in e44ed19. 

Now, we add a `commentId` property here:
https://github.com/WordPress/gutenberg/blob/e44ed1927d98e54acaa880e75051ce1a322e8781/packages/block-library/src/comment-template/util.js#L35

We have to keep the `commentId` property as the inner blocks like Comment Author Avatar, expect the property passed in the block context to be called `commentId`.